identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb

```diff
@@ -0,0 +1,246 @@
+   "source": [
+    "import os\n",
+    "import glob \n",
+    "import boto3\n",
+    "# import shutil\n",
+    "from PIL import Image \n",
+    "import PIL \n",
+    "\n",
+    "from pathlib import Path\n",
+    "from time import time\n",
+    "\n",
+    "# -------------------------------------------\n",
+    "mySession = boto3.session.Session()\n",
+    "aws_region = mySession.region_name\n",
+    "print(\"AWS Region: {}\".format(aws_region))\n",
+    "\n",
+    "# --- provide a bucket name ---\n",
+    "bucket_name = \"identity-verification-media-123\"\n",
+    "print(\"AWS Bucket: {}\".format(bucket_name))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad367ffe",
+   "metadata": {},
+   "source": [
+    "## Step 1. Create an S3 Bucket \n",
+    "----\n",
+    "The following will create a new S3 bucket which will store images and other objects that are used in subsequent modules. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9c1b5d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "\n",
+    "s3 = boto3.client('s3')\n",
+    "s3.create_bucket(Bucket=bucket_name,\n",
+    "                 CreateBucketConfiguration={\n",
+    "                     'LocationConstraint': aws_region})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa4eb367",
+   "metadata": {},
+   "source": [
+    "## Step 2. Download Labeled Faces in the Wild\n",
+    "------\n",
+    "\n",
+    "The Labeled Faces in the Wild is a database of face photographs designed for studying the problem of unconstrained face recognition. The data set contains more than 13,000 images of faces collected from the web. Each face has been labeled with the name of the person pictured. 1680 of the people have two or more distinct photos. More details can be found here: http://vis-www.cs.umass.edu/lfw/\n",
+    "\n",
+    "**NOTE: this may take a couple minutes to download** "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a74e1d7",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!rm -R lfw-funneled\n",
+    "!curl http://vis-www.cs.umass.edu/lfw/lfw-funneled.tgz -O\n",
+    "!tar -xvf lfw-funneled.tgz\n",
+    "!rm -R lfw-funneled.tgz"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99e353a9",
+   "metadata": {},
+   "source": [
+    "## Step 3. Convert images from jpg to png (or jpeg)\n",
+    "-----\n",
+    "Rekognition supports png and jpeg images, here we are going to read the \"jpg\" images, convert them to png and save them in a directory called idv_media. In this step we are simply extracting images that start with the letter \"A\". \n",
+    "\n",
+    "\n",
+    "**NOTE: this may take a minute or two to run**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c4721cdc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.getcwd()\n",
+    "\n"
+   ]
+  },
```

Comment:
What's the purpose of this code block? Is it needed?

Reply:
removed.